src/components/app-content-switcher/AppContentSwitcher.jsx

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
+import Switch from '@mui/material/Switch'
+import Stack from '@mui/material/Stack'
+
+const AppContentSwitcher = ({ isStudent, handleChange, tooltipRight, tooltipLeft, spacing, leftText, rightText, colorActive, colorInActive, typographyVariant, styles }) => {
+  const defaultStyles = {
+    display: 'flex',
+    flexDirection: 'row'
+  }
+
+  const renderLeftBlock = () => {
```

Comment:
You have 2 similar blocks, path array with objects and do two blocks with map

Reply:
ok if we need those tooltips, will do. but i think it will be hard to read

Reply:
refactored